src/ExtensionsSample/Samples/NotificationHubSamples.cs

```diff
@@ -36,7 +47,17 @@ public static class NotificationHubSamples
             messageProperties = "{\"message\":\"Hello\",\"location\":\"Redmond\"}";
         }
 
-        // The binding sends multiple push notification to any clients registered with the template
+        // NotificationHub binding out Notification type
+        // This binding broadcasts native apple push notification to all the registered iOS clients
+        // method successfully exits.
```

Comment:
I think here and throughout you're missing a "when" before "successfully exits". Recommend fixing up the wording on all these.